app/presenters/links_table_presenter.rb

```diff
@@ -46,8 +46,7 @@ def headers
         text: "Status",
       },
       {
-        text: "",
-        format: "numeric",
+        text: "<span class=\"govuk-visually-hidden\">Edit</span>".html_safe,
```

Comment:
The use of a visually hidden span looks correct in this place: https://github.com/alphagov/local-links-manager/pull/1369/files#diff-0c9c05c90dea6bde81d4427841d705abb0c2397c3ad7c9fc763f0d4b84e06a7cR17 but can you explain what this visually hidden `Edit` span does?

Reply:
This is the column header. The visual style idiom for these tables is that we don't show a column heading here (it would either be generic/not helpful like "Action" or it would just repeat the visual link, like "Edit"), so we don't want anything visible, but a blank column heading isn't helpful for screen readers, so we add a visually hidden column header. This appears consistent with how (eg) manuals publisher handles this.